lib/fizzy/execute.cpp

```diff
@@ -510,6 +510,13 @@ inline uint64_t popcnt64(uint64_t value) noexcept
     return static_cast<uint64_t>(__builtin_popcountll(value));
 }
 
+template <typename T>
+__attribute__((no_sanitize("float-divide-by-zero"))) inline constexpr T fdiv(T a, T b) noexcept
```

Comment:
The attribute disables UBSan check in both GCC and Clang. Using C++ attributes (`[[gnu::no_sanitize("")]]`) is also possible but requires some changes to compiler's warning options. So I think it is fine to leave it as is for now.